python-3.12.yaml

```diff
@@ -106,6 +116,8 @@ subpackages:
           # pyconfig.h is needed at runtime... ugh.
           mkdir -p "${{targets.destdir}}"/usr/include/python3.12
           mv "${{targets.subpkgdir}}"/usr/include/python3.12/pyconfig.h "${{targets.destdir}}"/usr/include/python3.12
+          mkdir -p "${{targets.subpkgdir}}"/usr/lib
+          mv ${{targets.destdir}}/usr/lib/libpython3.so "${{targets.subpkgdir}}"/usr/lib
```

Comment:
-dev packages must be co-installable too for compiled extensions.

Please move this to the -default sub package.

Reply:
I think I'm just going to remove it for now.